scripts/release.sh

```diff
@@ -6,5 +6,9 @@ set -euo pipefail
 DEFAULT_TAG=naushadh/hive-metastore:latest
 TAG=${TAG:-$DEFAULT_TAG}
 
-docker build --tag "$TAG" .
-docker push "$TAG"
+docker buildx create --name multiarch --use
```

Comment:
This causes an error when rerunning the script on the same machine. Can you wrap this with in a way it is safe to rerun? ex:

```suggestion
docker buildx ls | grep multiarch || docker buildx create --name multiarch --use
```

Reply:
Yep! I just tested a couple of repetitious runs on my machine with your suggestion and it was able to successfully run before and after `multiarch` is created.